### PR TITLE
FDP-255 ~ Fixes LMD bugs

### PR DESCRIPTION
**Pull request by kevinsmeets**

### CODE REVIEW

File: osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/application/mapping/Iec60870Mapper.java [hunk 1]
```diff
@@ -17,12 +19,26 @@
...
+  public Iec60870Mapper(final TimeZone timeZone) {
+    super(false);
+
+    this.timeZone = timeZone;
+  }
+
+  @PostConstruct
+  public void initialize() {
+    this.init();
+  }
```
Comment (smvdheijden) [inline osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/application/mapping/Iec60870Mapper.java:33]: Would this also work? 
```suggestion
  public Iec60870Mapper(final TimeZone timeZone) {
    super(false);
    this.timeZone = timeZone;
    this.init();
  }
```
If so, there would be no need to call the `initialize` method in `Iec60870AsduConverterTest` and other test classes.

File: osgp/protocol-adapter-iec61850/osgp-core-db-api-iec61850/src/main/java/org/opensmartgridplatform/core/db/api/iec61850/repositories/LmdDataRepository.java [hunk 2]
```diff
@@ -18,4 +19,7 @@
   LightMeasurementDevice findByDeviceIdentification(String deviceIdentification);
 
   List<LightMeasurementDevice> findByDigitalInputBetween(short start, short end);
```
Comment (smvdheijden) [inline osgp/protocol-adapter-iec61850/osgp-core-db-api-iec61850/src/main/java/org/opensmartgridplatform/core/db/api/iec61850/repositories/LmdDataRepository.java:21]: Is this method still needed?

File: osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/application/mapping/Iec60870Mapper.java (hunk 1 above)
Comment (kevinsmeets) [reply osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/application/mapping/Iec60870Mapper.java]: Seems to work fine, updated the code.

File: osgp/protocol-adapter-iec61850/osgp-core-db-api-iec61850/src/main/java/org/opensmartgridplatform/core/db/api/iec61850/repositories/LmdDataRepository.java (hunk 2 above)
Comment (kevinsmeets) [reply osgp/protocol-adapter-iec61850/osgp-core-db-api-iec61850/src/main/java/org/opensmartgridplatform/core/db/api/iec61850/repositories/LmdDataRepository.java]: Nope.

**Review by smvdheijden — APPROVED** [review]

**Pull request merged**